packages/react/index.ts

```diff
@@ -1,3 +1,9 @@
 export { extraEslintDependencies, reactEslintJson } from './src/utils/lint';
 export { CSS_IN_JS_DEPENDENCIES } from './src/utils/styled';
 export { assertValidStyle } from './src/utils/assertion';
+
+export { Schema as LibrarySchema } from './src/schematics/library/schema';
+export { Schema as ApplicationSchema } from './src/schematics/application/schema';
+export { Schema as ComponentSchema } from './src/schematics/component/schema';
+export { Schema as ReduxSchema } from './src/schematics/redux/schema';
+export { StorybookConfigureSchema } from './src/schematics/storybook-configuration/schema';
```

Comment:
minor detail, but would rename to `StorybookConfigurationSchema` :)

Reply:
yup, that's the plan after #3455 will be merged